main/firmware-linux/template.py

```diff
@@ -274,6 +283,13 @@
         "network",
         ["adaptec/starfire*"],
     ),
+    (
+        "ti-tas",
+        "Texas Instruments Amplifiers",
+        None,
+        "audio",
+        ["INT88*", "TAS2*", "TIAS2*", "ti/tas*"],
+    ),
```

Comment:
this one is annoying:
```
 INT8866RCA2.bin -> ti/tas2563/INT8866RCA2.bin
 TAS2XXX38A5.bin -> ti/tas2781/TAS2XXX38A5.bin
 TAS2XXX38A7.bin -> ti/tas2781/TAS2XXX38A7.bin
 TAS2XXX38A8.bin -> ti/tas2781/TAS2XXX38A8.bin
 TAS2XXX38B8.bin -> ti/tas2781/TAS2XXX38B8.bin
 TAS2XXX38B9.bin -> ti/tas2781/TAS2XXX38B9.bin
 TAS2XXX38BA.bin -> ti/tas2781/TAS2XXX38BA.bin
 TAS2XXX38BB.bin -> ti/tas2781/TAS2XXX38BB.bin
 TAS2XXX38BE.bin -> ti/tas2781/TAS2XXX38BE.bin
 TAS2XXX38BF.bin -> ti/tas2781/TAS2XXX38BF.bin
 TAS2XXX38C3.bin -> ti/tas2781/TAS2XXX38C3.bin
 TAS2XXX38CB.bin -> ti/tas2781/TAS2XXX38CB.bin
 TAS2XXX38CD.bin -> ti/tas2781/TAS2XXX38CD.bin
 TAS2XXX38D3.bin -> ti/tas2781/TAS2XXX38D3.bin
 TAS2XXX38D4.bin -> ti/tas2781/TAS2XXX38D4.bin
 TAS2XXX38D5.bin -> ti/tas2781/TAS2XXX38D5.bin
 TAS2XXX38D6.bin -> ti/tas2781/TAS2XXX38D6.bin
 TAS2XXX38DF.bin -> ti/tas2781/TAS2XXX38DF.bin
 TAS2XXX38E0.bin -> ti/tas2781/TAS2XXX38E0.bin
 TAS2XXX387D.bin -> ti/tas2781/TAS2XXX387D.bin
 TAS2XXX387E.bin -> ti/tas2781/TAS2XXX387E.bin
 TAS2XXX387F.bin -> ti/tas2781/TAS2XXX387F.bin
 TAS2XXX3870.bin -> ti/tas2563/TAS2XXX3870.bin
 TAS2XXX3880.bin -> ti/tas2781/TAS2XXX3880.bin
 TAS2XXX3881.bin -> ti/tas2781/TAS2XXX3881.bin
 TAS2XXX3882.bin -> ti/tas2781/TAS2XXX3882.bin
 TAS2XXX3884.bin -> ti/tas2781/TAS2XXX3884.bin
 TAS2XXX3886.bin -> ti/tas2781/TAS2XXX3886.bin
 ti/
 TIAS2781RCA2.bin -> ti/tas2781/TIAS2781RCA2.bin
 TIAS2781RCA4.bin -> ti/tas2781/TIAS2781RCA4.bin
```

there's technically `tas2563` and `tas2781`. but the 2563 name is just random numbers like the other, so i kept them together